frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx

```diff
@@ -45,11 +45,25 @@ export function ClusterDestroy(props: { isLoading: boolean; cluster: Cluster })
       isLoading={props.isLoading}
       loadingTitle={loadingTitle}
       loadingMessage={
-        <Trans
-          i18nKey={`${props.cluster?.status}.inprogress.message`}
-          values={{ clusterName: props.cluster?.displayName }}
-          components={{ bold: <strong /> }}
-        />
+        props.cluster?.status === ClusterStatus.destroying && isHybrid ? (
+          // t('destroying.ai.inprogress.message')
```

Comment:
leftovers

Reply:
this is not a leftover. It is for the i18n parser to pick up the translation key below as it is constructed programatically